### ISSUE TITLE
[BUG] minor spelling mistake (CZECH TRANSLATION)

### ISSUE THREAD

**Issue by SonicGamer803** [bloxstraplabs/bloxstrap#4127]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [X] I did not answer truthfully to all the above checkboxes.

### Bloxstrap Version

v2.8.2

### What problem did you encounter?

So basically on the czech translation of bloxstrap, texture quality and level 3, the "Nejvštší" is supposed to be "Největší" please fix this its a spelling mistake its unacceptable
![Snímek obrazovky 2024-12-27 170244](https://github.com/user-attachments/assets/39205fad-6f20-41ab-8f85-1670ec5796ec)

### Bloxstrap Log

```text
N/A
```
